### ISSUE TITLE
Unapproved and not disbursed loan with color of closed loan when reversed

### ISSUE THREAD

**Issue by MexinaD** [openMF/community-app#2949]
## Description
For the loan which was reversed from disbursal back to unapproved, it shows the color of inactive loan instead of unapproved loan.

## Steps to Reproduce
Create an active client: Client -> Create Client
Create a loan product: Admin->Product->Loan Product
Apply a loan of the product created to the created client: Clients -> View the client profile -> New Loan
Approve, disburse and make a repayment.
Then undo disbursal, in client profile the loan will show the color of inactive/closed loan
Also if you undo approval,  in client profile the loan will also show the color of inactive/closed loan

## Expected Behaviour
The loan to have the color of the state its in, blue for the loan waiting to be disbursed and yellow for the loan waiting to be approved.

## Actual Behaviour
Loan with the color of closed loans

## Settings
- Mifos X version: 18.3
- Browser used: Chrome

## Screenshots, if any
How it is seen in client profile
![unapproved](https://user-images.githubusercontent.com/8160209/39698246-c3e31c68-51fc-11e8-9d36-aeb50bafa2b2.png)

In loan profile
![loanview](https://user-images.githubusercontent.com/8160209/39698274-e5ca8f6e-51fc-11e8-9f03-c27dde0f997f.png)

**Comment by loveloud** [comment]
Hello, I would love to fix this. Any guidance on where I can start?

**Comment by santoshconflux** [comment]
@Hackingforlife , Did you set up Community app by fetching the code and following the instructions from below link?

https://github.com/openMF/community-app

**Comment by loveloud** [comment]
@santoshconflux, Yes, I have finished setting up the community app...Though i need further guidance  on actually working on this issue

**Comment by santoshconflux** [comment]
@Hackingforlife , Did you follow the description? Which step you didn't understand?

**Comment by loveloud** [comment]
On the steps to reproduce, right from the first step, I'm not sure where to make the changes in the code?

**Comment by NeetigyaPod** [comment]
Are these issues active (ones with hacktoberfest tags)? I am looking for some good first issues so can I work on this?
